snapcraft/_baseplugin.py

```diff
@@ -85,10 +85,11 @@ def __init__(self, name, options, project=None):
         self.project = project
         self.options = options
 
+        part_dir = name.replace('/', '\N{BIG SOLIDUS}')
```

Comment:
why don't we just make it a subdir? I mean, keep the slash. Like partdir = os.path.join(project.parts_dir, *name.split('/'))